index.md

```diff
@@ -90,6 +90,53 @@ Die Liste der Extensions findet sich in [der Readme des Docker-Images][docker-im
 Anschließend kann mit `make index.pdf` die PDF-Datei kompiliert werden.
 Unter Windows kann sich der entsprechende Befehl aus dem `Makefile` entnommen werden.
 
+### Windows
+
+Start powershell
+
+**If scoop is not installed**
```

Comment:
Some colons would be nice here, to indicate that these are meant as section heading.